frontend/src/bundles/account-settings/components/settings/styles.module.scss

```diff
@@ -89,3 +92,43 @@
     margin: var(--m-3) 0;
     background-color: var(--color-gray-200);
 }
+
+.dltButton {
+    justify-self: end;
+    margin-top: var(--m-3);
+}
+
+.icon {
+    padding-right: 5px;
+}
+
+@media (max-width: $md) {
+    .userProfile {
+        padding: var(--p-3);
+    }
+
+    .title {
+        font-size: var(--font-size-level-3-heading);
+    }
+}
+
+@media (max-width: $sm) {
+    .form {
+        grid-template-columns: 1fr;
+    }
+
+    .btnContainer > button {
+        padding: var(--p-1) var(--p-2);
+    }
+
+    .submitBtn,
+    .dltButton {
+        justify-self: center;
+        width: 100%;
+    }
+
+    .submitBtn > button,
+    .dltButton > button {
```

Comment:
Why don't you use just class on the button?

Reply:
Button from another component. I can't contact her otherwise.